src/objdetect/objdetect.h

```diff
@@ -110,6 +114,29 @@ CvStatus QRCodeDetector_setEpsX(QRCodeDetector self, double epsX);
 CvStatus QRCodeDetector_setEpsY(QRCodeDetector self, double epsY);
 CvStatus QRCodeDetector_setUseAlignmentMarkers(QRCodeDetector self, bool useAlignmentMarkers);
 
+// FaceDetectorYN
+CvStatus FaceDetectorYN_New(const char *model, const char *config, Size input_size, float score_threshold,
+                            float nms_threshold, int top_k, int backend_id, int target_id,
+                            FaceDetectorYN *rval);
+CvStatus FaceDetectorYN_NewFromBuffer(const char *framework, VecUChar buffer, VecUChar buffer_config,
+                                      Size input_size, float score_threshold, float nms_threshold, int top_k,
+                                      int backend_id, int target_id, FaceDetectorYN *rval);
+void     FaceDetectorYN_Close(FaceDetectorYNPtr self);
+CvStatus FaceDetectorYN_Detect(FaceDetectorYN self, Mat img, Mat *faces);
+CvStatus FaceDetectorYN_SetInputSize(FaceDetectorYN self, Size input_size);
+CvStatus FaceDetectorYN_SetScoreThreshold(FaceDetectorYN self, float score_threshold);
+CvStatus FaceDetectorYN_SetNMSThreshold(FaceDetectorYN self, float nms_threshold);
+CvStatus FaceDetectorYN_SetTopK(FaceDetectorYN self, int top_k);
```

Comment:
Here, I think we should also add getters such as `getInputSize ()`, `getNMSThreshold`, etc. https://docs.opencv.org/4.x/df/d20/classcv_1_1FaceDetectorYN.html#a68b6fb9bffbed0f3d5c104996113f247
```suggestion
CvStatus FaceDetectorYN_SetTopK(FaceDetectorYN self, int top_k);
```

Reply:
Ok will do it 👍

Reply:
done